client/src/app/ui/modules/import-list/components/via-backend-import-list/backend-import-list.component.html

```diff
@@ -137,7 +137,7 @@ <h2>{{ 'Preview' | translate }}</h2>
             </mat-select>
         </mat-form-field>
     </div>
-    <div>
+    <div style="margin-bottom: 5px">
```

Comment:
Not entirely sure but afaik our linter should complain about inline style definitions. Use a class instead if possible.